frontend/lib/ui/Form/Form.tsx

```diff
@@ -2,31 +2,105 @@ import * as React from "react";
 
 export interface FormProps extends React.FormHTMLAttributes<HTMLFormElement> {
   children: React.ReactNode;
+  skip?: string[];
 }
-export const Form = React.forwardRef<HTMLFormElement, FormProps>(({ children, ...formProps }, ref) => {
+
+type Dir = "up" | "down" | "first" | "last";
+
+export const Form = React.forwardRef<HTMLFormElement, FormProps>(({ children, skip, ...formProps }, ref) => {
   const innerRef: React.MutableRefObject<HTMLFormElement | null> = React.useRef<HTMLFormElement>(null);
 
-  React.useEffect(() => {
-    const submitButton = innerRef.current?.querySelector("button[type=submit]") as HTMLButtonElement | null;
+  const inputList = React.useRef<HTMLInputElement[]>([]);
+  const submitButton = React.useRef<HTMLButtonElement | null>(null);
+
+  const moveFocus = React.useCallback((dir: Dir) => {
+    let activeIndex = inputList.current.findIndex((input) => document.activeElement === input);
+    if (activeIndex === -1) {
+      activeIndex = 0;
+    }
+    let targetIndex = activeIndex;
+    switch (dir) {
+      case "up":
+        targetIndex = activeIndex - 1;
+        break;
+      case "down":
+        targetIndex = activeIndex + 1;
+        break;
+      case "first":
+        targetIndex = 0;
+        break;
+      case "last":
+        targetIndex = inputList.current.length - 1;
+        break;
+    }
+    if (targetIndex < 0) {
+      targetIndex = inputList.current.length - 1;
+    } else if (targetIndex >= inputList.current.length) {
+      targetIndex = -1; //end of the line
+    }
 
+    if (targetIndex >= 0) {
+      const next = inputList.current[targetIndex];
+      if (next) {
+        next.focus();
+        setTimeout(() => {
+          next.select();
+        }, 1);
+      }
+    } else {
+      submitButton.current?.focus();
+    }
+  }, []);
+
+  React.useEffect(() => {
     const handleKeyDown = (event: KeyboardEvent) => {
-      if (event.key === "Enter") {
-        if (event.shiftKey || document.activeElement === submitButton) {
-          event.preventDefault();
-          event.stopPropagation();
-          //ref.current.submit fails in testing environment (jsdom)
-          innerRef.current?.dispatchEvent(new Event("submit", { bubbles: true, cancelable: true }));
-        } else {
+      switch (event.key) {
+        case "ArrowUp":
+          if (event.shiftKey) {
+            moveFocus("first");
+          } else {
+            moveFocus("up");
+          }
+          break;
+        case "ArrowDown":
+          if (event.shiftKey) {
+            moveFocus("last");
+          } else {
+            moveFocus("down");
+          }
+
+          break;
```

Comment:
On the Recounted page, it's not longer possible to change the radio button by pressing arrow up/down.

Reply:
I can't reproduce this locally, can you describe or record the behavior you're experiencing?

Reply:
Can reproduce it on Firefox, can't reproduce on Chromium. Select either of the radio buttons on the Recounted page with the mouse. Then use (shift +) arrow up/down to select the other radio button.

On Chromium both on the [main preview ](https://kiesraad-abacus.pages.dev/elections/1/data-entry/1/recounted) and the [branch preview](https://34803521.kiesraad-abacus.pages.dev/elections/1/data-entry/1/recounted), arrow up/down and shift + arrow up/down let you select the othe radio button.

On Firefox, behavior is ... weird.
When you select "Ja", arrow up/down does nothing. Shift + arrow up/down lets you select the other radio button, but in reverse: up gets you from "ja" to "nee".
When you select "nee", arrow up does nothing, but the first arrow down scrolls down, the second arrow down selects "ja". Shift + arrow up does nothing, shift + arrow down lets you select "ja".

Reply:
It should be fixed now.

Reply:
Tested in Firefox, works for me!